ceph.spec.in

```diff
@@ -839,7 +839,7 @@ rm -rf %{buildroot}
 %{_sbindir}/ceph-create-keys
 %{_sbindir}/rcceph
 %dir %{_libexecdir}/ceph
-%{_libexecdir}/ceph/ceph_common.sh
+%{_libdir}/ceph/ceph_common.sh
```

Comment:
Should delete the line `%global _libexecdir %{_exec_prefix}/lib` at the top of the spec file, too.